pkg/query.go

```diff
@@ -145,23 +145,31 @@ func (qc *queryConfigStruct) transformQueryResult(columnTypes []*sql.ColumnType,
 
 		switch column_types[i].ScanType() {
 		case reflect.TypeOf(boolean):
-			values[i] = values[i].(bool)
+			if values[i] != nil {
```

Comment:
Thanks for your PR, can you please simplify the code and perform only one null check before the switch ?